examples/building_custom_algorithms/with_baselines_or_sim.py

```diff
@@ -14,10 +14,11 @@
 
 class MyOwnAlgorithm(AlgoBase):
 
-    def __init__(self, sim_options={}, bsl_options={}):
+    def __init__(self, sim_options={}, bsl_options={}, verbose=True):
```

Comment:
This is an example, no need to change it
(If you need to do quick tests, do them on a seperate file that you don't commit)